test/jake/jakelib/migrate-test.js

```diff
@@ -430,6 +433,49 @@ test('migrate:up runs only the next unrun migration', (temp) => {
     });
 });
 
+test('migrate:up <name> runs only the defined unrun migration', (temp) => {
+  const migrationsPath = `${temp}/migrations`;
+  const migrationFile1 = '001_one.js';
+  const migrationFile2 = '002_two.js';
+  const migrationData = `
+      exports.up = () => Promise.resolve();
+      exports.down = () => Promise.resolve();
+    `;
+
+  fs.writeFileSync(`${migrationsPath}/${migrationFile1}`, migrationData);
+
+  fs.writeFileSync(`${migrationsPath}/${migrationFile2}`, migrationData);
+
+  return assertExec(
+    `node ${KNEX} migrate:up ${migrationFile2} \
+    --client=sqlite3 \
+    --connection=${temp}/db \
+    --migrations-directory=${migrationsPath}`,
+    'run_migration_002'
+  ).then(({ stdout }) => {
```

Comment:
it is preferable to use async/await syntax

Reply:
done

Reply:
was this change pushed?

Reply:
You are right, thank you for mention this, I forgot to do this, now this is done. Please, let me know if there anything else.